README.md

```diff
@@ -84,6 +84,7 @@ Environment variables:
 | `SIGNED_URLS_REQUIRED` | Require URLs to Via's Python endpoints to be signed so that Via can only be used by something that has the URL signing secret. Public instances of Via should _not_ enable this. Private instances of Via (e.g. the LMS app's instance of Via) _should_ enable this | `true`
 | `VIA_HTML_URL` | The URL of the Via HTML instance to redirect to for proxying HTML pages | `https://viahtml.hypothes.is/proxy`
 | `VIA_SECRET` | The secret that must be used to sign URLs to Via's Python endpoints if `SIGNED_URLS_REQUIRED` is on |
+| `JSTOR_PDF_URL` | Full URL of the JSTOR API for article's PDFs. Only required on instances where this feature is enabled. |
```

Comment:
The other env vars in this list are sorted in alphabetical order, can you move this accordingly.